server/routes/barcode/FindRecipientController.test.ts

```diff
@@ -0,0 +1,79 @@
+import { Request, Response } from 'express'
+import type { FindRecipientForm } from 'forms'
+import { SessionData } from 'express-session'
+import FindRecipientController from './FindRecipientController'
+import PrisonRegisterService from '../../services/prison/PrisonRegisterService'
+
+const req = {
```

Comment:
I think there is a better library - `jest-express` - that perhaps we can use to test express controllers etc in a much nicer manner, but this will do for now. 

I might create a ticket to look at implementing `jest-express` as I think it will improve our test coverage massively and will make for testing controllers like this much easier. One for another day .... 👍